roles/system/glusterfs_lv/lvchange/tests/vars.yml

```diff
@@ -0,0 +1,19 @@
+---
+# vars file for lvchange
+lvchange_list:
+- "{{ lv1 }}"
+- "{{ lv2 }}"
+lv1:
+  state: change
```

Comment:
'state' var needn't be specific to each lv.
state: change - will be same for all lvs, hence can be declared just once.